src/Game/UI/Controls/StbTextBoxCombo.cs

```diff
@@ -80,18 +87,27 @@ protected override void OnMouseUp(int x, int y, MouseButtonType button)
 				return;
 			}
 
-			
-			if (x > _arrow.X && x < _arrow.X + _arrow.Width && y > _arrow.Y && y < _arrow.Y + _arrow.Height)
+			if (_arrow != null)
 			{
-				_arrow.Hue = 0;
-				TextboxComboContextMenu contextMenu = new TextboxComboContextMenu(_items, Width, 100, _font, selectedIndex => SelectedIndex = selectedIndex)
+				_arrow.Graphic = ARROW_UP;
+				if (x > _arrow.X && x < _arrow.X + _arrow.Width && y > _arrow.Y && y < _arrow.Y + _arrow.Height)
```

Comment:
Hmm hmm hmm, I've seen this quite a bit, Maybe `Gump` should have a public method `bool Contains(Point point)`.

Reply:
I actually considered this.  This is a pretty standard function for game development.  I peeked in the `Utilities` folder to see if i could find anything, but no luck.  I definitely agree on this.

Reply:
Upon further investigation the GumpPic has its own event for mouse down and mouse up.  Instead of doing the hitbox check i just subbed to the events on the GumpPic control itself and then the hitbox test is no longer needed.

Reply:
subbed to the mouse events and moved the logic there. no more need for hitbox detection.